test/integration_tests/cli/diff_command_test.dart

```diff
@@ -143,5 +143,25 @@ void main() {
       },
       timeout: integrationTestTimeout,
     );
+
+    test(
+      'diffing cloud_firestore 4.3.1 to latest works',
+      () async {
+        // just some random package for testing the diff command for pub refs without version
```

Comment:
I saw that we are repeating ourselves with this comment of 'just some random package ...'. What do you think about adding this comment at the top of the test file instead of repeating it for every test? I know that it will (probably) be harder to understand where the package came from but I don't think that repeating it for all the tests is much better too :/